app/jobs/twilio_job.rb

```diff
@@ -1,6 +1,10 @@
 class TwilioJob  < ActiveJob::Base
   queue_as :default
 
+  rescue_from(Twilio::REST::RequestError) do
+    retry_job wait: 30.seconds
+  end
```

Comment:
Sidekiq will retry it 25 times (default)
https://github.com/mperham/sidekiq/wiki/Error-Handling#automatic-job-retry
cc: @steveyken